osu.Game.Tests/Visual/Gameplay/TestSceneStoryboardSamplePlayback.cs

```diff
@@ -57,7 +57,7 @@ public void TestStoryboardSamplesStopOnSkip()
 
         private void checkForFirstSamplePlayback()
         {
-            AddUntilStep("storyboard loaded", () => Player.Beatmap.Value.StoryboardLoaded);
+            AddAssert("storyboard loaded", () => Player.Beatmap.Value.Storyboard != null);
```

Comment:
Just going to note that this assertion means something slightly different now. Since the backing storage of `Storyboard` is a `Lazy<T>`, and the old `StoryboardLoaded` would (eventually) check `IsValueCreated`, this assertion now has changed from "was the storyboard loaded by something else" to "force load this storyboard, if not loaded already, and make sure it's there and also not null" - by the virtue of the getter of `Storyboard` accessing the `Lazy<T>`'s `Value`.

Not sure it's a big deal - just pointing out as I go along. Semi-related to the removal unused of `IsThingLoaded` members from `IWorkingBeatmap`. Unsure if anyone would ever actually need to query the creation state of those properties in context of a real usage (nobody sure had to up until now).

Reply:
i saw this, but am willing to wait for a failure before changing it further.

there's already a wait step for player loading so i think it should be okay.